packages/sit-onyx/src/components/OnyxSidebar/OnyxSidebar.vue

```diff
@@ -86,6 +85,9 @@ const isModalOpen = computed<boolean>({
   },
   set: (newVal: boolean) => {
     _isModalOpen.value = newVal;
+    if (!newVal) {
+      emit("close");
```

Comment:
Should we also switch to v-model:open for the Sidebar? Can be done in a follow up PR ofc